src/utils.js

```diff
@@ -8,7 +8,7 @@ function unixStylePath(filePath) {
 
 var PLUGIN_NAME = require('../package.json').name;
 
-var urlRegex = /^(https?|webpack(-[^:]+)?):\/\//;
+var urlRegex = /^(https?|webpack(-[^:]+)?|file):\/\//;
```

Comment:
```suggestion
var urlRegex = /^(https?|webpack(-[^:]+)?|file?):\/\//;
```
all of them were optional prior, should it be so still?

Reply:
Not exactly. The "?" in "https?" is only for the optional trailing "s", to support both "http" and "https".
In the webpack fragment the "?" applies only to the final part, not to the "webpack" prefix.
And the regex should match only in one of the three cases is present.